### PR TITLE
Added page with help about Search filters

### LINKED ISSUE #58: Add documentation for message search predicates
Message search predicates (introduced in https://github.com/Chatterino/chatterino2/pull/1237, https://github.com/Chatterino/chatterino2/pull/2634) are only usable if you know they exist. Providing documentation would help improve discoverability of this feature. Documentation page should include:
- All currently supported keywords, and their syntax
- Examples of search queries
- (Ideally) Screenshots of a "reference message history" and screenshots of the results produced by specific search queries

### PR DESCRIPTION
**Pull request by zneix**
Added information about predicate filters that we have in chatterino search popup and some general information about it.

Closes #58
Initial work for Chatterino/chatterino2#2671

### CODE REVIEW

**Comment by pajlada** [comment]
Good start! For final version, images should be uploaded to the repository

**Comment by pajlada** [comment]
I think it would be good to have a 'Added in version XXX' for each filter

**Review by MrAuro — APPROVED** [review]
SeemsGood TwitchTV Emoticon

File: docs/Search.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Search
+
+Chatterino has Search Popup which lets you quickly find messages in the chat. You can open it with the `Ctrl+F` shortcut.
```
Comment (talneoran) [inline docs/Search.md:3]:
```suggestion
Chatterino has a Search Popup which lets you quickly find messages in the chat. You can open it with the `Ctrl+F` shortcut or through the split menu.
```

Not sure If I'm right about the naming of the split menu but you can change it.

File: docs/Search.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Search filters
+
+You can use several search filters to narrow down your search:
+
+- `from:<username>` - shows messages only from certain username. Can be multiple usernames separated with a comma (added in 2.1.4)
+- `has:<flags>` - shows messages only containing specified elements, list of flags (added in 2.1.4):
+    - `link` - filters messages with links
+- `in:<channel>` - (useful in `#/mentions` channel) shows messages that originate from certain channels (added in 2.1.4)
```
Comment (talneoran) [inline docs/Search.md:12]:
```suggestion
- `in:<channel>` - (useful in `#/mentions` channel) shows messages that originate from certain channels (added in 2.3.0)
```

Maybe worth mentioning the option of comma seperated values here too? Or possibly once for all filters?

File: docs/Search.md (hunk 1 above)
Comment (zneix) [reply docs/Search.md:3]: I've been considering adding this, but I wanted to keep it short and simple to not overwhelm users with the amount of text there. I know I provided only 1 option out of 2 that are available, but honestly i feel like just being able to Ctrl+F the popup is enough to know about.

File: docs/Search.md (hunk 2 above)
Comment (zneix) [reply docs/Search.md:12]: I agree I've been a little inconsistent with the comma separation, I'll just make a note that all predicates can have comma-separated values :+1:

Comment (zneix) [reply docs/Search.md]: Both resolved in fc3f0c2

**Review by pajlada — APPROVED** [review]

**Pull request merged**